docs/release-notes/october-2024.md

```diff
@@ -0,0 +1,130 @@
+---
+slug: october-2024
+title: October 2024
+hide_title: true
+description: >-
+    Release notes include the changes, fixes, and additions in specific versions of Semgrep.
+tags:
+  - Release notes
+---
+
+# Semgrep release notes for October 2024
+
+## 🌐 Semgrep AppSec Platform
+
+### Added
+
+- Added a **Scan details** page and drawer for all completed scans. Use this page to troubleshoot or view information about individual scans. 
```

Comment:
Do we want to explain what we mean by drawer?

Reply:
I'll check our trusty google style guide if it has a preferred or more common term!

Reply:
They prefer calling it a notification drawer, which doesn't match our use case.
https://developers.google.com/style/word-list#notification-drawer

I'll stick to calling it a sliding pane: https://developers.google.com/style/ui-elements